### PR TITLE
Newsweek milestone1

### PR DESCRIPTION
**Pull request by prolajumokeoni**
# Newsweek-Milestone1
In this milestone:
> We created the header and footer
> We used bootstrap classes for styling
> We used bootstrap flex and grid for the header and footer 
>  We  made the header and footer responsive for mobile, medium, and desktop using bootstrap classes
#### We have implemented all changes requested from the previous review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  <title>Newsweek</title>
+</head>
+<body>
+    <header class="main-header d-flex justify-content-between align-items-center">
+      <div class="d-none d-xl-block p-30">
+        <h5 class="date text-white">Wed, Jan 20, 2021</h5>
+      </div>
+      <div>
+        <img class="d-none d-xl-block mb-4" src="./images/svgexport-2.svg" alt="logo">
+        <img class="main-logo d-xl-none" src="./images/svgexport-2.svg" alt="logo">
```
Comment (teekaytech) [inline index.html]: The Newsweek logo has a link. i.e. use the `<a>`  tag around the logo.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        <a class="nav-link text-body" href="#"><h5>Business</h5></a>
+        <a class="nav-link text-body" href="#"><h5>Tech & Science</h5></a>
+        <a class="nav-link text-body" href="#"><h5>Culture</h5></a>
+        <a class="nav-link text-body" href="#"><h5>Newsgeek</h5></a>
+        <a class="nav-link text-body" href="#"><h5>Sports</h5></a>
+        <a class="nav-link text-body" href="#"><h5>Health</h5></a>
+        <a class="nav-link text-body" href="#"><h5>Experts</h5></a>
+        <a class="nav-link text-body" href="#"><h5>Vantage</h5></a>
+        <button type="button" class="btn ">Search <span><i class="fas fa-search"></i></span></button>
+      </nav>
```
Comment (teekaytech) [inline index.html]: This should be a form tag with the search field, not a button.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                </svg> 
+            </div>
```
Comment (teekaytech) [inline index.html]: This Newsweek logo (svg image) should have a link.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          </div>
+      
+          <div class="col-md-5 col-xs-12">
+            <ul class="footer-1-socials d-flex align-items-center justify-content-md-end">
+              <li> <i class="fa fa-facebook-square"></i> </li>
+              <li> <i class=" mx-2 fa fa-twitter"></i> </li>
+              <li> <i class=" mx-2 fa fa-instagram"></i> </li>
+              <li> <i class=" mx-2 fa fa-twitter"></i> </li>
+              <li> <i class=" mx-2 fa fa-pinterest"></i> </li>
+            </ul>
```
Comment (teekaytech) [inline index.html]: Social media icons should have a link.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @prolajumokeoni & @blakbox23 :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- Descriptive PR title and summary :+1: 
- Project files are well organized into directories :ok_hand: 
- Linter checks running/passing successfully :100: 

__However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_**One of the major implementation requirements for this project and all its milestones is the use of Bootstrap classes everywhere for styling and responsiveness. Therefore, this review will focus on this aspect.**_

- [x] Use less of custom CSS rules. Rules like margins, padding, list-style, etc all have their corresponding bootstrap classes. Please research and use them.
- [x] You can also increase the font-weight of the top navbars, as well as the `LOGIN` and `subscribe` texts on top of the navbar, just to make them look similar to the original page.
- [x] Observe the difference between the images below and make necessary adjustments. Pay special attention to the font size and the spaces in the red box.
Tablet View: 
Original page:
original
Clone page:
clone

- [x] Please check my inline comments below and fix them as appropriate.



* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html (hunk 4 above)
Comment (prolajumokeoni) [reply index.html]: We have effected all changes @teekaytech, thank you for your review

**Review by elisha2kyakpo1 — APPROVED** [review]
🎉Congratulations guys, You can now move.
I'm super happy to see you make all the changes requested.💪✔
Just in case you have a question, reach me on slack by typing elisha kyakopo in the search bar

**Comment by prolajumokeoni** [comment]
> 🎉Congratulations guys, You can now move.
> I'm super happy to see you make all the changes requested.💪✔
> Just in case you have a question, reach me on slack by typing elisha kyakopo in the search bar

Thank you for your review @elisha2kyakpo1

**Pull request closed without merging**